02/app.js

```diff
@@ -1,28 +1,38 @@
-document.addEventListener('DOMContentLoaded', init);
+document.addEventListener("DOMContentLoaded", init);
 
 function init() {
-    const clickEl = document.querySelector('.error--click');
-    const enterEl = document.querySelector('.error--enter');
+  const clickEl = document.querySelector(".error--click");
+  const enterEl = document.querySelector(".error--enter");
+  const alert = document.querySelector(".alert");
 
-    setRandomPosition(clickEl);
-    setRandomPosition(enterEl);
+  setRandomPosition(clickEl);
+  setRandomPosition(enterEl);
 
-    initEventWithError(clickEl, 'click', new RangeError('Błąd zakresu!'));
-    initEventWithError(enterEl, 'mouseenter', new TypeError('Błąd typu!'));
+  initEventWithError(clickEl, "click", new RangeError("Błąd zakresu!"));
+  initEventWithError(enterEl, "mouseenter", new TypeError("Błąd typu!"));
 
+  alert.addEventListener("click", function () {
+    alert.classList.add("alert--hidden");
+  });
 }
 
 function setRandomPosition(element, error = null) {
-    element.style.top = Math.random() * 600 + 'px';
-    element.style.left = Math.random() * 800 + 'px';
-
-    if(error) {
-        throw error;
+  try {
+    element.style.top = Math.random() * 600 + "px";
+    element.style.left = Math.random() * 800 + "px";
+    if (error) {
+      throw error;
```

Comment:
Przechwytywanie błędu zaraz w miejscu jego rzucania, nie jest najlepszym pomysłem.
Chodzi o to, że przechwytywać możemy gdzieś "dalej", bo inaczej wystarczyłby nam sam `if`.
Dlatego należy to zrobić co najmniej w miejscu uruchomienia funkcji - w tym przypadku linia 36.